cf/functions.py

```diff
@@ -3174,6 +3429,9 @@ def environment(display=True, paths=True):
         "dask": _get_module_info("dask"),
         # Then Python libraries not related to CF
         "netCDF4": _get_module_info("netCDF4"),
+        "h5netcdf": _get_module_info("h5netcdf"),
+        "h5py": _get_module_info("h5py"),
+        "s3fs": _get_module_info("s3fs"),
```

Comment:
We should update the docstring example output in line with these new additions.

Reply:
Done: bd45bda11c77ff82bb6d5b59f2c8a4974af31db5

Reply:
Nice, thanks. Marking as resolved.